acmw-db-app/pages/_app.js

```diff
@@ -1,13 +1,62 @@
 import '../styles/globals.css';
 import Head from 'next/head';
+import NavBar from '../components/NavBar';
+import React, { useEffect, useState } from 'react';
+
+const loadingWheel = (
+  <div className="lds-ring"><div></div><div></div><div></div><div></div></div>
```

Comment:
can you indent this for clarity
this is so minor I'm going to approve anyways bc everything else looks good

Reply:
bruh its literally just a bunch of empty divs to make a circle that i stole from online do i have to

Reply:
okay fine but it made me scared